### DISCUSSION TITLE
Contributor list is not updated anymore

### DISCUSSION THREAD

**Discussion by Technoboy-** [community/community#16889] [Discussions]
We find that https://github.com/apache/pulsar contributor list is not updated for almost several days.
Seems it updated today, but the data is incorrect.

Eg:  I have contributed about 116 commits, but only 107.  The commit lines is also incorrect.

**Comment by a49a** [comment]
+1 
https://github.community/t/project-not-update-contributor-datas/
I have the same problem.

**Comment by MrKrishnaAgarwal** [comment]
Hey
Checkout This Website: https://support.github.com/request?tags=docs-generic
